conda/base/context.py

```diff
@@ -414,6 +412,11 @@ def post_build_validation(self):
             errors.append(error)
         return errors
 
+    @property
+    def plugin_manager(self):
```

Comment:
```suggestion
    def plugin_manager(self):
    """
    This is the preferred way of accessing the ``PluginManager`` object for this application.
    This property appears here to avoid problems with cyclical imports elsewhere in the code.
    """
     
```